### ISSUE TITLE
small error in volume level example

### ISSUE THREAD

**Issue by mckaygerhard** [BashGui/easybashgui#19]
i have small error with code example of volume meter, only happened rare times! and i dont know why i checked the lline and i cant found any issue!

### code example

```
#!/usr/bin/env bash

source easybashgui

adjust "Please, set Volume level" 15 40 75
```

### output error

```
(standard_in) 1: syntax error
(standard_in) 2: syntax error
/home/general/Devel/easybashgui/./easybashgui.lib: line 811: [: -le: unary operator expected
```

### information

* bash : GNU bash, version 5.2.15(1)-release (x86_64-pc-linux-gnu)
* easybashgui: git lasted 20240202
* coreutils:  9.1
* supermode="yad"
* yad 0.40
* GTK: using gtk 3.33

**Comment by mckaygerhard** [comment]
ok i can reproduce.. its raised when you cancel the script


https://github.com/BashGui/easybashgui/assets/1571724/e3c4aafc-5f14-4adf-bf1d-2fd0387e55b3

**Comment by vaisarger** [comment]
Wow! Bug found!

This is bug fix:
`		${dialogo} --title "${supertitle:-EasyBashGUI}: adjust" --scale --text "${testo_zenity}" --value ${init_value_percent} --min-value ${start_value_percent} --max-value ${end_value_percent} ${dimensione_finestra_1} ${dimensione_finestra_2} \
		2> /dev/null 1> "${dir_tmp}/${file_tmp}"
		#exit_if_user_closes_window
		exit_code="${?}"
		uscita="${exit_code}"
		#
		if [ ${uscita} -eq 0 ]
			then
			#
			#
			value_percent=$(tr -dc '[[:digit:]]' 0< "${dir_tmp}/${file_tmp}" )
			: value_percent
			#
			final_value_temp=$(echo "scale=2; ${value_percent} * ${step}" | bc -l )
			final_value_float=$(echo "scale=2; ${start_value} + ${final_value_temp}" | bc -l )
			: final_value_float
			#
			final_value=$(arrotonda "${final_value_float}" )
			: final_value
			#
			echo "${final_value}" 1> "${dir_tmp}/${file_tmp}" 
			#
		elif [ ${uscita} -ne 0 ]
			then
			#clean_temp`

I'll fix it in next release... ;)

**Comment by vaisarger** [comment]
Fixed in version 12.0.5.

**Comment by vaisarger** [comment]
Fixed in version 12.0.5

**Comment by vaisarger** [comment]
Fixed in version 12.0.5.

**Comment by mckaygerhard** [comment]
fixed at commit 86138088f6995cf41dfa89d3a49af7a2d6898678 thanks

**Comment by vaisarger** [comment]
You're welcome ☺️

Il giorno sab 17 feb 2024 alle ore 02:43 Герхард PICCORO Lenz McKAY <
***@***.***> ha scritto:
